index.html

```diff
@@ -2922,10 +2922,10 @@ <h2>Thing Model</h2>
 
     </section>
 
-    <section id="wot-profiles">
+    <section id="wot-profiles" class="informative">
```

Comment:
You are right, there is no reason to change a section to informative just to refer to non-normative references in the text. However, I think we should be aligned with the Chapter 7 structure: E.g., Binding Templates and Scripting API sections are also informative. Since Profile has published a [first working draft so far](https://www.w3.org/TR/wot-profile/) so far, I think it makes sense for this section to be informative as well.